test/e2e/cluster.sh

```diff
@@ -159,8 +159,6 @@ else
     SKIP_AFTER_UPGRADE="${SKIP_AFTER_SCALE_DOWN}|${SKIP_AFTER_UPGRADE}"
   elif [ "${SCALE_CLUSTER}" = "true" ]; then
     SKIP_AFTER_UPGRADE="${SKIP_AFTER_SCALE_DOWN}"
-  else
```

Comment:
I see what you did here

Reply:
This is [SC2269](https://github.com/koalaman/shellcheck/wiki/SC2269): "This variable is assigned to itself, so the assignment does nothing."